tempo/kfserving/k8s.py

```diff
@@ -242,3 +258,49 @@ def _get_spec(self, model_spec: ModelSpec) -> dict:
     def to_k8s_yaml_spec(self, model_spec: ModelSpec) -> str:
         d = self._get_spec(model_spec)
         return yaml.safe_dump(d)
+
+    def list_models(self, namespace: Optional[str] = None) -> Sequence[RemoteModel]:
+        self.create_k8s_client()
+        api_instance = client.CustomObjectsApi()
+
+        if namespace is None and self.runtime_options is not None:
+            namespace = self.runtime_options.k8s_options.namespace
+
+        if namespace is None:
+            return []
+
+        try:
+            models = []
+            response = api_instance.list_namespaced_custom_object(
+                group="serving.kubeflow.org",
+                version="v1beta1",
```

Comment:
Would it be limited to just v1beta1? We could add issue to extend to the earlier crds

Reply:
yes at present